addons/website_slides_survey/models/slide_slide.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class SlidePartnerRelation(models.Model):
+    _inherit = 'slide.slide.partner'
+
+    user_input_ids = fields.One2many('survey.user_input', 'slide_partner_id', 'Certification attempts')
```

Comment:
Actually I don't really like this field as there are access rights limitations and I think it is not really useful. In order to use it you will have to be in sudo mode or have survey officer / manager group. I think we can achieve current result without the o2m (I will update / check others occurrences of it).

Reply:
Common you know you like it.